jquery.flot.touchNavigate.js

```diff
@@ -14,175 +14,369 @@
     }
 
     function initTouchNavigation(plot, options) {
-        var prevPanX = 0,
```

Comment:
I'd like to see these consolidate in a gectureState object that is passed to all the functions that need to update it